### DISCUSSION TITLE
Does this library support SSR?

### DISCUSSION THREAD

**Discussion by alexbjorlig** [simeydotme/svelte-range-slider-pips#98] [Q&A] [answered]
Trying to use with Sveltekit, get this error:

```
<RangeSlider> is not a valid SSR component. You may need to review your build config to ensure that dependencies are compiled, rather than imported as pre-compiled modules. Otherwise you may need to fix a <RangeSlider>.
```

**Comment by simeydotme** [comment]
Hmm,

I haven't tried this, but there's no way to have it purely SSR without Hydration as all the interaction is JS.
If anyone has experience executing this please let me know, otherwise I'll see what I can do in the coming weeks :)

**Reply by simeydotme** [reply, marked as answer]
Seems it does now! see https://github.com/simeydotme/svelte-range-slider-pips/issues/101
